src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -379,18 +379,21 @@
 {%- endblock form_rows -%}
 
 {%- block widget_attributes -%}
+    {{- " " -}}
     id="{{ id }}" name="{{ full_name }}"
     {%- if disabled %} disabled="disabled"{% endif -%}
     {%- if required %} required="required"{% endif -%}
     {{ block('attributes') }}
 {%- endblock widget_attributes -%}
 
 {%- block widget_container_attributes -%}
+    {{- " " -}}
```

Comment:
shouldn't this be moved inside the next "if"?

Reply:
I don't think so.

The output of this `{{ widget_attributes }}` ended on my example with like `<divid ... >`